lib/bnchmrkr.rb

```diff
@@ -125,38 +116,41 @@ def slowest_overall
   # +b+ Symbol that represents a known lambda
   # +mode+ :fastest, :slowest, :mean, :median, :total
   # return boolean if a is faster than b, false if invalid
-  def is_faster?(a, b, mode = :total)
-    result = calculate_per_lambda
-    return false unless result.has_key?(a) and result.has_key?(b)
-    result[a][mode] < result[b][mode]
+  def is_faster?(a, b, mode = :real)
```

Comment:
`is_mark_faster?` might be an interesting method, that way we can look for the fastest run her, and we can look for the fastest aggregate run in a different way